packages/components/src/components/dropdown-list/dropdown-list.tsx

```diff
@@ -172,34 +215,12 @@ const DropdownList = React.forwardRef((props, ref) => {
     );
 
     if (portal_id) {
```

Comment:
@niloofar-deriv `portal_id` type is `string`, Either this if check is redundant or the type should be optional.

Reply:
I'll made the type optional